backend/tournaments/migrations/0001_initial.py

```diff
@@ -0,0 +1,134 @@
+                ('suspended_until', models.DateField(help_text='Until what date the user is suspended from games.', null=True, blank=True)),
+                ('reliability', models.IntegerField(help_text='Reliability rating.')),
+                ('preferred_hours', models.CharField(default='yyyyyyyyyyyyyyyyyyyyyyyy', max_length=24, help_text='24-character string of format "yyyyyrrrrwwwwwwwwyyyyyyy", UTC-based.')),
+                ('preferred_control', models.CharField(default='45 45', max_length=100, help_text='Comma separated list of ‘MM SS’ time controls.')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Round',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, primary_key=True, auto_created=True)),
+                ('start', models.DateTimeField(help_text='When the round starts.')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Tournament',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, primary_key=True, auto_created=True)),
+                ('short_name', models.SlugField(max_length=32, help_text='E.g. monthly15, sb4, etc.')),
+                ('name', models.CharField(max_length=160, help_text='Snail Bucket Monthly 2015')),
+                ('type_code', models.CharField(max_length=32, help_text='Tournament type code, e.g. "RR", "Swiss", etc.')),
+                ('signup_start', models.DateTimeField(help_text='When registration is open.')),
+                ('signup_end', models.DateTimeField(help_text='When registration is no longer open.')),
+                ('games_start', models.DateTimeField(help_text="Round 1 won't start before that time")),
+                ('rounds_count', models.IntegerField(help_text='Number of rounds in the tournament.')),
+                ('rounds_start_cron', models.CharField(max_length=32, help_text='Starting date of every round, in crontab format.')),
+                ('min_bucket_size', models.IntegerField(help_text='Minimum number of players in a bucket.')),
+                ('max_bucket_size', models.IntegerField(help_text='Maximum number of players in a bucket.')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='TournamentPlayer',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, primary_key=True, auto_created=True)),
+                ('fixed_rating', models.IntegerField(help_text="Fixed raiting. If NULL, it means that the bot haven't filled the rating yet", null=True, blank=True)),
+                ('member', models.ForeignKey(to='tournaments.Tournament')),
+                ('tournament', models.ForeignKey(to='tournaments.Tournament', related_name='players')),
+            ],
+        ),
+        migrations.AddField(
+            model_name='round',
```

Comment:
This is hard to follow. Can we somehow make the whole table defined in a single place without AddField?